tests/sclT0009-unused-symbols/sym01.exec

```diff
@@ -0,0 +1,2 @@
+# format: exec
+exec:
```

Comment:
> Tests are falling because there is abs path of your home + working directory in reference file.

Oh god, this is embarrassing :/ 

@jaa127 I'll refine my solution following your advice, but in the new PR, as @hrj suggests.

@hrj I'm going to take the tests out of this PR, and create a new one for the tests.